extras/codespell-ignore-words-list.txt

```diff
@@ -0,0 +1 @@
+unkown
```

Comment:
@aentinger I want to bring this to your attention. It is required due to these enum values:
https://github.com/107-systems/107-Arduino-TMF8801/blob/4475c053e15b0d6a59200ff0fb7e38c6aadb25f2/src/TMF8801/TMF8801_Status.h#L30-L38
I assume this is a misspelling of "unknown", but wasn't sure what the best approach would be for correcting this (i.e., does the misspelled value need to be retained in addition to the correct one for backwards compatibility?).

So I just took the easy way out and added it to the ignore list.

Reply:
Hi 👋 you are right, it's a misspelling of unknown. I'm going ahead and merge this PR (thank you very much for the contribution) and will correct in a follow-up PR.